app.js

```diff
@@ -3,7 +3,13 @@ const app = express();
 const port = 3000;
 
 app.get("/", (req, res) => {
-  res.send("Hello World from chris!");
+  res.send(
+    "<h1>Carefully watch your thoughts, for they become your words. Manage and watch your words, for they will become your actions. ~ Mahatma Gandhi ~ </h1><a href='/info'>Info page</a>"
```

Comment:
Maybe start to look at views/templates for rendering pages. Also set up your static public pages.

https://expressjs.com/en/starter/static-files.html